src/components/PlayModel.js

```diff
@@ -0,0 +1,110 @@
+import React, { useState } from "react";
+import { MdClose } from "react-icons/md";
+import { usePlaylistContext } from "../context/PlayListContext";
+import Toast from './ToastComp';
+
+
+function PlayModel({ video }) {
+  const [playlistName, setplaylistName] = useState("");
+
+  const {
+    toggleModal,
+    display,
+    setDisplay,
+    playlist,
+    setPlaylist,
+    addPlaylists,
+    removeFromPlaylist,
+    addToPlaylist,
+  } = usePlaylistContext();
+  return (
+    <div style={{ display: display }} className="modal--container">
+      <div id="myModal" className="modal">
+        <div className="modal-content">
+          <MdClose onClick={toggleModal} className="close"></MdClose>
+
+          <div className="modal-body">
+            <p className="modal-title">Add to Playlist</p>
+            <div className="divider"></div>
+
+            {playlist?.map((playlist) => {
```

Comment:
You have mapped playlist and inside it in callback you have given the same name playlist, have different name in callback so that if you have any error later on it would be easy to debug.